py/qqa/webpages/tables.py

```diff
@@ -23,7 +23,11 @@ def write_nights_table(outfile, exposures):
     )
     template = env.get_template('nights.html')
 
-    html = template.render(nights=np.unique(exposures['NIGHT']))
+    nights_concat = np.unique(exposures['NIGHT'])
+    nights_sep = [[str(night), str(night)[0:4], str(int(str(night)[4:6])-1), str(night)[6:]] for night in nights_concat]
```

Comment:
I think jinja supports dictionaries which would make this more readable and robust in the future, e.g. making a list of `dict(night=night, year=str(night)[0:4], ...)` and then the template can be something like
```
startDate : new Date({{night.year}}, {{night.month}}, {{night.day}}),
```
Also, confirming: the calendar needs months as 0-indexed, while days are 1-indexed?

Reply:
yes, I don't know why thats the case, but months start at 0 and days start at 1.